docs/cvl/expr.md

```diff
@@ -242,6 +243,36 @@ approximation is {term}`sound`: it won't cause violations to be hidden.  See
 {ref}`grounding` for more detail.
 ```
 
+* The prover is capable of calculating the `sum` of ghost mappings over
+specified indices. For example, if we have a ghost mapping `ghost
+mapping(address => mapping(bytes32 => mathint)) myGhost`, and want to sum all
+the values of the ghost over all addresses for a given `bytes32` value `b`, one
+can write `sum address a. myGhost[a][b]`. This returns a `mathint`-typed
+number that sums all known values of `myGhost` over the first index.
+The full syntax for this is `sum type1 t1, type2 t2, ... typeN tN.
+ghostName[...]`. See [here](https://github.com/Certora/Examples/blob/master/CVLByExample/Summarization/GhostSummary/GhostSums/README.md)
```

Comment:
@yoav-el-certora fyi - we are hard-coding the link to the example on `master` in the docs here (even though it hasn't been merged). This breaks if the files are moved. Please ensure to keep the example in this location.